lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -245,6 +248,9 @@ export default function ContentSelector({
         />
       );
       break;
+    case 'youtube':
+      dialog = <YouTubePicker onCancel={cancelDialog} onSelectURL={() => {}} />;
```

Comment:
I think I'll leave it as `noop` for now, to avoid having to test this behavior.

Once I get to it then I will switch and actually test that path.

Reply:
Not sure if this is about the URL format. 

Via accepting youtube URLs now might be only useuful for testing (and/or enabling it for the public Via). We might also add `video://` URLs on the LMS side and Via will parse those too.

Reply:
Currently, this component will generate a URL in the form of `youtube://{videoId}`, but it's just because I had to put something there 😅 

It can be easily changed. It's not being used anywhere yet.

That's also why I prefer to not add any behavior to this component yet, as I think we probably need to align this first.